### ISSUE TITLE
How can i summary some information in wandb

### ISSUE THREAD

**Issue by reniew** [wandb/client#300]
* Weights and Biases version: latest 
* Python version: 3.6
* Operating System: MacOS

### Description
I used tensorflow estimatore and logging some metric and sub loss value
But metric and sub loss value doesn't show in wandb summary

### What I Did

In Estimator model function
```
        logging_hook = tf.train.LoggingTensorHook({"vae_loss": vae_loss,
                                                "prediction_loss": self.prediction_loss,
                                                'kl_divergence': kl_divergence,
                                                'autoencoder_loss': autoencoder_loss,
                                                'accuracy': accuracy_for_train},
                                                every_n_iter=1000)

        return tf.estimator.EstimatorSpec(mode = mode,
                                        loss = self.loss,
                                        eval_metric_ops = self.metrics,
                                        train_op = self.train_op,
                                        training_hooks = [logging_hook],
                                        predictions = {'prediction': self.prediction})
```
and start estimator

    estimator.train(dataset.train_input_function, hooks=[wandb.tensorflow.WandbHook(steps_per_log=1000)])

**Comment by vanpelt** [comment]
Hey @reniew I'm not sure what's happening with the hook.  The easiest way to collect tensorflow metrics is to pass `tensorboard=True` to your call to wandb.init.  That will automatically collect all metrics logged to tensorboard and add a new tensorboard tab to your run page!  Can you give this a try and let me know if it works?  https://docs.wandb.com/docs/frameworks/tensorflow.html
